development/button-app/pages/button/index.mdx

```diff
@@ -27,43 +38,76 @@ It uses following tailwind classes for providing styles: ** ``` w-1/2 bg-red-400
 ```
 
 ## Sample Code Demo
-
+This is the sample styles included when you return **ReuseButton** as shown in the demo code. It has the features of setting the **busy state** and the **busy text** which will be visible when in **busy state**.
 <Sample/>
 
 ```js path=../../components/button/Sample name=Sample
 ```
 
 ## Outline Button Demo
-
+This is a simple ReuseButton with the styling of **border-2 border-blue-600 bg-transparent font-bold text-blue-600 py-1 px-2** to give it a **transparent button** look with outlines. It can be suitable for situations where a more subdued visual appearance is desired while still maintaining the interactive nature of a button.
 <OutlineButton/>
 
 ```js path=../../components/button/OutlineButton name=OutlineButton
 ```
 
 ## Success Sample Button Demo
-
+The Success Sample Button is a customizable button that provides feedback to the user by showing a **loading** state for a specified duration and then disabling the button and displaying a **success** state. The **success** state and the **busy** state along with the **duration** can be customised as per the requirement.
```

Comment:
customizable -> customised 
Remove this part as the success state button can not be imported to change, They have to copy the code and write their own component.